docs/community/README.md

```diff
@@ -0,0 +1,132 @@
+# How to Get Involved
+
+<!-- TODO: what is community, what are the touchpoints -->
+
+The Knative community consists of three different audiences:
+
+![Diagram of Knative audiences: users, developers, operators, and contributors](images/knative-audience.svg)
+
+* **Developers** write serverless and event-driven applications using the
+  constructs surfaced by Knative.  They build containers and applications which
+  leverage the concepts exposed by Knative to deliver services to end users and
+  systems.
+
+* **Operators** build and maintain Kubernetes platforms where Knative is
+  installed.  They also manage the default settings for Knative on those
+  clusters.
+
+* [**Contributors**](./contributing.md) are the authors of the Knative
+  components; they contribute and review source code, run the community
+  meetings, document and publicize the project, and otherwise keep the project
+  running.
+
+Knative components are intended to operate well stand-alone, or integrated into
+a larger platform offering by either an internal systems team or a cloud
+provider.  You can see a [list of organizations that have adopted Knative in our
+community repo](https://github.com/knative/community/tree/main/ADOPTERS.MD).
+
+## Questions
+
+For the fastest response, you can ask questions on the [`#general`, `#serving`,
+`#eventing`, or `#functions` channels of the Knative
+Slack](https://slack.knative.dev/).
+
+We also have a [Stack Overflow topic,
+`knative`](https://stackoverflow.com/questions/tagged/knative) (as well as
+`knative-serving`, `knative-eventing`, and `knative-functions`), and the
+[knative-users](https://groups.google.com/forum/#!forum/knative-users) mailing
+list if you prefer those formats.
+
+## Bug Reports and Feature Requests
+
+Knative is composed of many different components. We use GitHub Issues to track
+bug reports and feature requests. While it's most helpful if you know the
+specific component that the bug is happening in, we understand that it can be
+difficult to tell sometimes. A clear bug report in the wrong component is much
+better than a partial report in the right component, so feel free to file your issue in one of these three main repositories if you're not sure:
+
+* [Serving](https://github.com/knative/serving/issues/new?template=bug-report.md)
+* [Eventing](https://github.com/knative/eventing/issues/new?template=bug.md)
+* [Functions](https://github.com/knative/func/issues/new)
+
+A good bug report should include:
+
+- What you were trying to do, and what happened
+- What version of Knative and Kubernetes you are using (if using a cloud provider, indicate which one)
+- Relevant resource yaml, HTTP requests, or log lines
+
+## Community Meetups
```

Comment:
just noticed we use basically the same text here and on the contributing page... we should probably create a snippet so as not to repeat ourselves.

That would make a good first issue, so I think we can skip updating now and I'll open an issue for it